packages/imperative/src/cmd/src/yargs/YargsConfigurer.ts

```diff
@@ -80,14 +80,19 @@ export class YargsConfigurer {
                         rootCommandName: this.rootCommandName,
                         commandLine: this.commandLine,
                         envVariablePrefix: this.envVariablePrefix,
-                        promptPhrase: this.promptPhrase
+                        promptPhrase: this.promptPhrase,
                     }).invoke({ arguments: argv, silent: false, responseFormat: this.getResponseFormat(argv) })
                         .then((_response) => {
                             Logger.getImperativeLogger().debug("Root help complete.");
                         })
                         .catch((rejected) => {
-                            process.stderr.write("Internal Imperative Error: Root command help error occurred: "
-                                + rejected.message + "\n");
+                            const daemonStream = ImperativeConfig.instance.daemonContext?.stream;
+                            if(daemonStream) {
+                                daemonStream.write(DaemonRequest.create({ stderr:`Internal Imperative Error: Root command help error occurred: ${rejected.message}\n`}));
+                            } else {
+                                process.stderr.write("Internal Imperative Error: Root command help error occurred: "
```

Comment:
I'm not too concerned about these two lines. 
But if we want, we could copy the new test added and not define the daemonStream in the mocked ImperativeConfig 😋